GitHub-Actions/two-stage-pipeline-template/{{cookiecutter.outputDir}}/.github/workflows/pipeline.yaml

```diff
@@ -54,20 +54,22 @@ jobs:
     if: github.event_name == 'push'
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - run: |
           # trigger the tests here
 
   delete-feature:
     if: startsWith(github.event.ref, 'feature') && github.event_name == 'delete'
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
       - uses: aws-actions/setup-sam@v2
 
       - name: Assume the testing pipeline user role
-        uses: aws-actions/configure-aws-credentials@v1
```

Comment:
Can we add a comment to explain why 16 is used here? Like reference of https://github.com/aws-actions/configure-aws-credentials#notice-node12-deprecation-warning

Reply:
Yes. That's why. Version 12 is deprecated. Where should I Put the link?

Reply:
link added